<launch>
        <node name = "leftmcu_node" pkg = "robotswitch_controller" type = "left_mcu_node" respawn="false" output="screen" >
                <param name="left_port_"  value="/dev/ttyUSB1"/>
                <param name="left_baud_"  value="115200"/>
        </node>
</launch>